builtin/providers/google/resource_google_project.go

```diff
@@ -25,28 +23,54 @@ import (
 // }
 func resourceGoogleProject() *schema.Resource {
 	return &schema.Resource{
+		SchemaVersion: 1,
+
 		Create: resourceGoogleProjectCreate,
 		Read:   resourceGoogleProjectRead,
 		Update: resourceGoogleProjectUpdate,
 		Delete: resourceGoogleProjectDelete,
+
 		Importer: &schema.ResourceImporter{
 			State: schema.ImportStatePassthrough,
 		},
+		MigrateState: resourceGoogleProjectMigrateState,
 
 		Schema: map[string]*schema.Schema{
 			"id": &schema.Schema{
 				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"project_id": &schema.Schema{
```

Comment:
Ahh yes. `d.Id()` and `d.Get("id")` don't seem to return anything during create, even if they're set in the manifest. We were [testing with an ID](https://github.com/hashicorp/terraform/blob/master/builtin/providers/google/resource_google_project_test.go#L448), but that was being ignored as [project was being assigned](https://github.com/hashicorp/terraform/blob/master/builtin/providers/google/resource_google_project.go#L64) from the provider. So as far as I can tell, `id` appears to be reserved for state only. Does that sound correct?